lib/ublksrv.c

```diff
@@ -296,10 +296,15 @@ static void ublksrv_submit_fetch_commands(struct _ublksrv_queue *q)
 	__ublksrv_queue_event(q);
 }
 
-static int ublksrv_queue_is_done(struct _ublksrv_queue *q)
+static inline int __ublksrv_queue_is_done(const struct ublksrv_queue *tq)
 {
-	return (q->state & UBLKSRV_QUEUE_STOPPING) &&
-		!io_uring_sq_ready(&q->ring);
+	return (tq_to_local(tq)->state & UBLKSRV_QUEUE_STOPPING) &&
+		!io_uring_sq_ready(&tq_to_local(tq)->ring);
+}
+
+int ublksrv_queue_is_done(const struct ublksrv_queue *tq)
+{
+	return __ublksrv_queue_is_done(tq);
```

Comment:
Hello, 

It should be more readable by adding local variable:

```
const struct _ublksrv_queue *q = tq_to_local(tq);
return __ublksrv_queue_is_done(q);
```

Thanks,

Reply:
done